src/shorthands/JBox.jl

```diff
@@ -0,0 +1,126 @@
+function _JBox(
+    cornerpoint1::Point,
+    cornerpoint2::Point,
+    color::String,
+    action::Symbol,
+    vertices::Bool,
+)
+    sethue(color)
+    verts = box(cornerpoint1, cornerpoint2, action, vertices = vertices)
+    return verts[2]
+end
+function _JBox(points::Array, color::String, action::Symbol, vertices::Bool)
+    sethue(color)
+    verts = box(points, action, vertices = vertices)
+    return verts[2]
+end
+function _JBox(
+    pt::Point,
+    width::Real,
+    height::Real,
+    color::String,
+    action::Symbol,
+    vertices::Bool,
+)
+    sethue(color)
+    box(pt, width, height, action, vertices = vertices)
+    return Point(pt.x - width / 2, pt.y + height / 2)
+end
+function _JBox(
+    pt::Point,
+    width::Real,
+    height::Real,
+    cornerradius::Float64,
+    color::String,
+    action::Symbol,
+)
+    sethue(color)
+    box(pt, width, height, cornerradius, action)
+    return Point(pt.x - width / 2, pt.y + height / 2)
+end
+
+"""
+    JBox(cornerpoint1::Point, cornerpoint2::Point; color="black", action=:stroke, vertices=false)
+
+Create a box (rectangle) between two points and do an action.
+Use vertices=true to return an array of the four corner points: bottom left, top left, top right, bottom right.
```

Comment:
This seems to be off. It only returns `top left` now

Reply:
I think that was in the issue? https://github.com/Wikunia/Javis.jl/pull/356#discussion_r663476254

Reply:
Yeah just the docstring needs to be updated

Reply:
Oh sorry.

Reply:
BTW Is this necessary? Since box() will return all the points of the box as a  list which seems much better.

Reply:
It's useful for using the `pos` function.